org.eclipse.xtext/src/org/eclipse/xtext/generator/trace/node/GeneratorNodeProcessor.xtend

```diff
@@ -42,38 +42,103 @@ class GeneratorNodeProcessor {
 		}
 	}
 	
-	@Accessors protected static class Context {
```

Comment:
do we need a getLines to stay api compatible or is this no api? (afaik package is still exported as xfriends/internal

Reply:
I was thinking about the compatibility, but it seems like the GeneratorNodeProcessor API is not used very often and it's marked as internal.

Reply:
yes thus the question if if you want to keep the other places of compatibility code

Reply:
This would mean to track the access to `getLines` members otherwise it will break the runtime compatibility. I think it's better to just replace the implementation. It's a protected Class in an internal package with experimental API.

Reply:
ok